schema/fair.js

```diff
@@ -95,6 +95,17 @@ const FairType = new GraphQLObjectType({
     organizer: {
       type: FairOrganizerType,
     },
+    has_published_organizer_profile: {
+      type: GraphQLBoolean,
+      resolve: ({ organizer }) => {
+        const id = organizer && organizer.profile_id;
+        if (id === null) return false;
+        return gravity(`profile/${id}`)
+        .then(profile => {
+          return profile && profile.published;
+        });
```

Comment:
Nit: This could either do with some indentation so it’s a level higher than the `return` statement, or maybe just even make it one line: `return gravity(`profile/${id}`).then(profile => profile && profile.published)`